packages/uswds-utilities/src/styles/rules/border-radius.scss

```diff
@@ -3,17 +3,17 @@
 border-radius
 ----------------------------------------
 usage:
-  .border-radius-[key]
+  .radius-[key]
 ----------------------------------------
 output:
   border-radius: [value];
 ----------------------------------------
 example:
-  .border-radius-0 {
+  .radius-0 {
     border-radius: 0; }
 
-  .border-radius-pill {
-    border-radius: 10em; }
+  .radius-pill {
+    border-radius: 99rem; }
```

Comment:
```suggestion
    border-radius: 99rem; 
  }
```

Reply:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)